docs/apis-tools/spring-zeebe-sdk/getting-started.md

```diff
@@ -162,3 +162,44 @@ public void handleJobFoo(final ActivatedJob job) {
 ```
 
 See [the configuration documentation](/apis-tools/spring-zeebe-sdk/configuration.md) for a more in-depth discussion on parameters and configuration options for job workers.
+
+## Writing test cases
+
+You can start up an in-memory test engine and do assertions by adding this Maven dependency:
+
+```xml
+<dependency>
+  <groupId>io.camunda</groupId>
+  <artifactId>spring-boot-starter-camunda-test</artifactId>
+  <version>8.6.x</version>
+  <scope>test</scope>
+</dependency>
+```
+
+Note that **the test engine requires Java version >= 21**. If you cannot run on this Java version, you can use [Testcontainers](https://www.testcontainers.org/) **instead**. Testcontainers require that you have a Docker installation locally available on the developer machine. Use this dependency:
+
+```xml
+<!--
+  Alternative dependency if you cannot run Java 21, so you will leverage Testcontainer
+  Make sure NOT to have spring-zeebe-test on the classpath in parallel!
+-->
+<dependency>
+  <groupId>io.camunda</groupId>
+  <artifactId>spring-boot-starter-camunda-test-testcontainer</artifactId>
+    <version>8.6.x</version>
+  <scope>test</scope>
+</dependency>
+```
+
+Then, start up the test engine in your test case by adding `@ZeebeSpringTest`
+
+```java
+@SpringBootTest
+@ZeebeSpringTest
+public class TestMyProcess {
+  // ...
+```
+
+An example test case is [available here](https://github.com/camunda-community-hub/camunda-cloud-examples/blob/main/twitter-review-java-springboot/src/test/java/org/camunda/community/examples/twitter/TestTwitterProcess.java).
+
+> Please do not use `zeebeTestEngine.waitForBusyState(...)` to wait for a timer. This will not work as this is also triggered by an incoming job activation.
```

Comment:
:x: We should add this content to the [Zeebe Process Test docs](https://docs.camunda.io/docs/apis-tools/java-client/zeebe-process-test) instead of the Spring SDK docs. Just add a link in the Spring SDK docs. The Zeebe Process Test docs contain information about writing tests.  

With 8.7, Camunda Process Test will become the default testing library and we should change the link to the Camunda Process Test docs.    

We can keep the content in the Zeebe Process Test docs for users that still use Zeebe Process Test.

Reply:
Thank you for that suggestion, will apply this!

Reply:
Addressed with aca17e2f5e548d5287c4b19a2ec3c7c7b6f57a9b